docs/src/pages/start/pick-quasar-flavour.md

```diff
@@ -21,22 +19,31 @@ Here's a quick comparison:
 | Include Quasar from public CDN                                                             | **Yes** | -          | -                             |
 | Build SPA, PWA                                                                             | **Yes** | **Yes**    | **Yes**                       |
-| Build SSR (+ optional PWA client takeover)                                                 | -       | **Yes**    | Yes*                          |
-| Build Mobile Apps, Electron Apps                                                           | **Yes** | **Yes**    | Yes*                          |
-| Build Browser Extensions                                                            | -       | **Yes**    | Yes*                          |
+| Build SSR (+ optional PWA client takeover)                                                 | -       | **Yes**    | Yes(*)                          |
+| Build Mobile Apps via Cordova or Capacitor                                                 | **Yes** | **Yes**    | Yes(*)                          |
+| Develop Mobile Apps with HMR directly on your phone.                                       | -       | **Yes**    | Yes(*) |
+| Build Desktop Apps via Electron                                                            | -       | **Yes**    | Yes(*)                             |
+| Build Browser Extensions                                                                   | -       | **Yes**    | Yes(*)                          |
 | Quasar **App Extensions**                                                                  | -       | **Yes**    | - |
-| App icons & splash screens easy management through [Icon Genie](/icongenie/introduction) | - | **Yes** | - |
+| Easy management of App icons & splash screens via [Icon Genie](/icongenie/introduction)    | - | **Yes** | - |
 | Dynamic RTL support for Quasar components                                                  | **Yes** | **Yes**    | **Yes**                       |
 | Generating your own website/app RTL equivalent CSS rules automatically by Quasar           | -       | **Yes**    | **Yes**                       |
 | **Ensure everything "simply works" out of the box**, using latest and greatest Quasar specs.   | -       | **Yes**    | -                             |
 | **Tight integration between build modes**, taking full advantage of all Quasar's capabilities. | -       | **Yes**    | -                             |
-| Develop mobile apps with HMR directly on your phone.                                       | -       | **Yes**    | Yes*                          |
-| One codebase to rule SPA, PWA, SSR, Mobile Apps, Electron Apps                             | -       | **Yes**    | Yes*                          |
+| One codebase to create SPA, PWA, SSR, Mobile Apps, Electron Apps                             | -       | **Yes**    | Yes(*)                      |
```

Comment:
Don't forget BEX 😋

Reply:
BEX is in there. :)

Reply:
`One codebase to create SPA, PWA, SSR, Mobile Apps, Electron Apps` ? Am I losing my mind?

Reply:
I'm thinking `One codebase to create SPA, PWA, SSR, Mobile Apps, Electron Apps, Browser Extensions`

Reply:
Got it. Updated.